test/bigtest/network/config.js

```diff
@@ -670,7 +671,7 @@ export default function config() {
     return {};
   });
 
-  this.get('/custom-labels', {
+  this.get('/kb-credentials/:id/custom-labels', {
```

Comment:
there are already GET and PUT `/kb-credentials/:id/custom-labels` routes defined below
we need both `/custom-labels` and `/kb-credentials/:id/custom-labels` routes because first one is used in e-holdings to fetch custom labels for a KB that current user has access to and second one is to get custom labels for a particular KB configuration